shaders/compute_nbody/slang/particle.frag.slang

```diff
@@ -0,0 +1,37 @@
+/* Copyright (c) 2025, Sascha Willems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Sampler2D samplerColorMap;
+Sampler2D samplerGradientRamp;
+
+struct VSOutput
+{
+    float4 Pos : SV_POSITION;
+    float GradientPos;
+    float2 CenterPos;
+    float PSize : SV_PointSize;
+    // remove
+    float PointSize;
```

Comment:
Would, instead of `PSize` and `PointSize` something like
`float PointSize : SV_PointSize;`
be enough?

Reply:
Will check. With Slang from the latest SDK, using SV_PointSize in the fragment shader results in a crash for me.